calico_node/startup/autodetection/reachaddr.go

```diff
@@ -0,0 +1,50 @@
+package autodetection
+
+import (
+	"fmt"
+	gonet "net"
```

Comment:
I don't buy the latter point - there is always a risk that if a library is named the same thing as golang native library people will assume it's the native library.  Say I didn't need go-net in here, I wouldn't need to rename the cnet one so it'd just be net...

Personally, I like the idea of always prefixing the native ones with go.  It makes it clearer that there is an identically named package *and* it is an easy naming convention to stick to.

That said, I'm not too fussed either way.  Ping me if it's really irritating you.